### DISCUSSION TITLE
Elastalert: command alert no write permissions

### DISCUSSION THREAD

**Discussion by ITDutchie** [Security-Onion-Solutions/securityonion#3638] [Unsupported Versions]
Hello, 

Currently Im looking into the possibilities with elastalert. I see there is an option to write the output to a file by using the command option in the options from alert. The alerts.txt file (file where the alert needs to write to) is placed in /opt/so/rules/elastalert/playbook. 

I have the following commands in the rule: 

alert:
- "modules.so.playbook-es.PlaybookESAlerter"
- "command"
command: "tee -a alerts.txt"

This is a pretty basic command. I see the file is accessible from the so-elastalert docker. But i got a message in stderr that there is not enough persmissions to perform the task. Following the error:

tee: alerts.txt: Permission denied

Is this by design? Or is there a way make this possible?

**Comment by ITDutchie** [comment]
Anyone? :)

**Reply by jertel** [reply]
I've not attempted to do what you're asking, or understand what you're overall goal is. But if you are trying to have Elastalert output a file then I suggest you have it output to /tmp/, and then `docker cp so-elastalert:/tmp/alerts.txt /tmp`. You can then access the alerts.txt output file on the host's /tmp dir.
